testing/tests/DevExpress.ui.widgets/overlay.tests.js

```diff
@@ -3285,15 +3288,93 @@ QUnit.test("overlay should render with correct z-index by default", function(ass
 });
 
 QUnit.test("base z-index should be changed using the static method", function(assert) {
-    Overlay.baseZIndex(10000);
+    try {
+        Overlay.baseZIndex(10000);
 
-    var $overlay = $("#overlay").dxOverlay({
-            visible: true
-        }),
-        overlay = $overlay.dxOverlay("instance"),
-        $content = $(overlay.content()),
-        $wrapper = $content.parent();
+        var $overlay = $("#overlay").dxOverlay({
+                visible: true
+            }),
+            overlay = $overlay.dxOverlay("instance"),
+            $content = $(overlay.content()),
+            $wrapper = $content.parent();
+
+        assert.equal($content.css("zIndex"), 10001, "z-index for content is correct");
+        assert.equal($wrapper.css("zIndex"), 10001, "z-index for wrapper is correct");
+    } finally {
+        Overlay.baseZIndex(1500);
```

Comment:
Why in this test`try/finally` is used, but not the `afterEach` as here https://github.com/DevExpress/DevExtreme/pull/6424/files#diff-a02b83188316c95a58b33ccf1ce50850R132 ?

Reply:
It because baseZIndex redefined in this test only but not in each test

Reply:
As I can see, a lot of the previous tests does not modify `baseZIndex` too, but their `afterEach` handler resets the `baseZIndex` anyway. Maybe it's better to use `try/finally' everywhere or extract the separate module?